### DISCUSSION TITLE
Vercel still running commit checks on repo it was removed from

### DISCUSSION THREAD

**Discussion by MichaelNey** [vercel/community#1487] [Help] [answered]
### Summary

Hello, on Vercel I removed the Git repo from a project, and on GitHub I also removed the app Vercel. So it shouldn't have access to the repo, correct?

When a user in my repo made a PR, Vercel still managed to make a commit check. For other PRs after I removed Vercel it didn't seem to do any commit checks. Only in this case.

![image](https://user-images.githubusercontent.com/25756024/217923526-68fd1231-b1a3-42a9-a99c-ac7cd53065ea.png)

I have no GitHub apps in the Org, or Repo level. The Vercel project also does not have a Git repo linked.

Thoughts? We have our own github actions now, but they seem to have been not run because of this Vercel commit check...

### Additional information

![image](https://user-images.githubusercontent.com/25756024/217923771-8842921d-702c-43ad-a1d7-e5a9693e7602.png)
![image](https://user-images.githubusercontent.com/25756024/217923814-ece32178-2263-4c9e-ad17-961510dc71f8.png)
![image](https://user-images.githubusercontent.com/25756024/217923895-acef2853-8eb8-495e-ab00-a19f43c1bc4e.png)

Based on this, I would expect for Vercel to have 0 access to my repo, and not do any of these checks.

**Comment by samcx** [comment, marked as answer]
@MichaelNey Hi there! There may be a stray project where it is connected to. What does it show when you click on the Details button? Can you also share with us the commit SHA made on that :pr: ?

**Reply by MichaelNey** [reply]
Looks like the commit was made before Vercel was removed, but the branch seems to have been pushed after which is confusing. This could just be a weird timing issue I didn't see. Otherwise future commits haven't triggered it so this doesn't block us which is good.

ed29e87f5d8f0344ee6fc00e17881c06bb46ec4a
